recipe/meta.yaml

```diff
@@ -22,12 +22,12 @@ requirements:
   host:
     - python
     - pip
-    - cython  # [not win]
+    - cython 0.29.32  # [not win]
```

Comment:
is cython not used on Windows?

Reply:
Yep

Reply:
But let me double check the reason

Reply:
Yes, that's correct https://github.com/pydantic/pydantic/blob/a84cfacf9caf08bef8edca076c3325c3cf49475d/requirements.txt#L3